docs/content_model.md

```diff
@@ -89,33 +89,39 @@ They represent three categories of Content Types that you can create
 
 ### Content browser
 
-If you want to get an overview of the content currently in your website, go to the **Content** tab
-and select **Content structure** or **Media**.
-The **Browse** option on the left allows you to browse through all the content of the respective category.
+During your work with eZ Platform you might need to select content from the Repository.
+This happens for example when you want to move or copy a Content item, embed an image, link two Content items, etc.
+In such cases you use the **Content Browser**.
+
+To access the **Content Browser**, go to the **Content** tab and select **Content structure** or **Media**.
+Then, on the left side of the screen, click the **Browse** icon.
 
 ![Content browser in the menu](img/left_menu_browse.png "Content browser in the menu")
 
-User accounts, which are also considered content in eZ Platform, are viewable in the **Users** tab accessible in **Admin** (see [Users](site_organization/organizing_the_site.md#users)).
+With the **Content Browser**, you can switch between the Grid, Panels and Tree views to navigate through the content of the site.
+Depending on your permissions, you might be able to review the regular content, media, forms, site skeletons and User accounts.
+You can also use the **Content Browser** to search the Repository for content, select content options, create Bookmarks, and create new content when needed.
```

Comment:
```suggestion
You can also use the **Content Browser** to search the Repository for content, select content options, create bookmarks, and create new content when needed.
```
"select content options" is unclear.

Reply:
I am changing "select content options" to "edit content"